ACE/ace/Atomic_Op.h

```diff
@@ -304,6 +304,7 @@ class ACE_Export ACE_Atomic_Op<ACE_Thread_Mutex, unsigned long>
   ACE_Atomic_Op (unsigned long c);
   ACE_Atomic_Op (const ACE_Atomic_Op<ACE_Thread_Mutex, unsigned long> &c);
   ACE_Atomic_Op<ACE_Thread_Mutex, unsigned long> &operator= (unsigned long rhs);
+  ACE_Atomic_Op<ACE_Thread_Mutex, unsigned long> &operator= (const ACE_Atomic_Op<ACE_Thread_Mutex, unsigned long> &c);
```

Comment:
Some test extension for this operation? With C++11 we do prefer std::atomic, we should be careful extending ACE_Atomic_Op, not that we add all kinds of ACE extensions which make it harder later for users to migrate to std::atomic